openvpn/session/manager.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package session
+
+import (
+	"github.com/mysterium/node/identity"
+	"sync"
+	"github.com/mysterium/node/session"
+)
+
+// NewManager returns session manager which maintains a map of session id -> session
+func NewManager(m session.Manager) *manager {
```

Comment:
What is the purpose of this "session" manager inside openvpn package? we already have one.

Reply:
Doc updated, this manager is kind of decorator for underlying session manager to account for openvpn clientID